client/landing/stepper/declarative-flow/internals/steps-repository/shared-components/setup-form/index.tsx

```diff
@@ -79,39 +86,37 @@ const SetupForm = ( {
 				selectedFile={ selectedFile }
 			/>
 			<FormFieldset>
-				<FormLabel htmlFor="link-in-bio-input-name">{ __( 'Site name' ) }</FormLabel>
+				<FormLabel htmlFor="setup-form-input-name">{ __( 'Site name' ) }</FormLabel>
 				<FormInput
-					name="link-in-bio-input-name"
-					id="link-in-bio-input-name"
+					name="setup-form-input-name"
+					id="setup-form-input-name"
 					value={ siteTitle }
 					onChange={ onChange }
-					placeholder={ __( 'My Link in Bio' ) }
+					placeholder={ __( 'My Site Name' ) }
 					isError={ invalidSiteTitle }
 				/>
 				{ invalidSiteTitle && (
 					<FormInputValidation
 						isError
-						text={ __( `Oops. Looks like your Link in Bio doesn't have a name yet.` ) }
+						text={ __( `Oops. Looks like your site doesn't have a name yet.` ) }
```

Comment:
Is it ok to change the copy here and in the other places? (like `description` instead of `biography`, ... )

Reply:
Good point. A lot of thought and discussion went into the copy for each of these flows. Rather than go generic, it would be easy enough to make this dynamic. A simple approach would be to have each flow provide its translated strings to the form as props, while also having some sane defaults in case those are missing?

Reply:
Yes, I think that would be really useful in this PR 👍

Reply:
@escapemanuele - This has been done. Let me know if you have any concerns with the implementation.